src/components/ProgramCard.tsx

```diff
@@ -147,26 +147,27 @@ const ProgramCard: FC<ProgramCardProps> = ({
             </Stack>
           )}
         </Stack>
-
-        <Stack>
-          <Typography
-            typography="captionregular"
-            color="neutral400.main"
-            mb="4px"
-          >
-            {selectedLanguage?.includes('English')
-              ? 'Starting From'
-              : 'Mulai Dari'}
-          </Typography>
-
-          <Typography
-            typography="subheading2"
-            color="warning100.main"
-            minHeight="26px"
-          >
-            {!program.ctPrice ? '-' : program.ctPrice}
-          </Typography>
-        </Stack>
+        {isVoucherRedeem ? (
+          <Stack>
+            <Typography
+              typography="captionregular"
+              color="neutral400.main"
+              mb="4px"
+            >
+              {selectedLanguage?.includes('English')
+                ? 'Starting From'
+                : 'Mulai Dari'}
+            </Typography>
+
+            <Typography
+              typography="subheading2"
+              color="warning100.main"
+              minHeight="26px"
+            >
+              {!program.ctPrice ? '-' : program.ctPrice}
```

Comment:
atau kalo mau hemat 1 karakter 🐧 
```suggestion
              {program.ctPrice ? program.ctPrice : '-' }
```